charts/galoy/values.yaml

```diff
@@ -1,106 +1,576 @@
+    ##
+    probes:
+      ## Enable/disable API probes for healthcheck
+      enabled: true
+
+      ## If pod is starting up/healthy
+      liveness:
+        initialDelaySeconds: 60
+        periodSeconds: 10
+        failureThreshold: 5
+        timeoutSeconds: 1
+
+      ## When to expose the pod to the service
+      readiness:
+        initialDelaySeconds: 5
+        failureThreshold: 5
+        successThreshold: 2
+        timeoutSeconds: 1
+
+  ## Configuration values for Trigger components.
+  ##
+  trigger:
+    ## Override the components name (defaults to trigger).
+    ##
+    nameOverride:
+
+    ## Number of replicas.
+    ##
+    replicas: 1
+
+    ## Log Level
+    ##
+    logLevel: debug
+
+    ## Port on which Exporter Server is running
+    ##
     port: 8888
-cron:
-  - name: cronjob
-    appEntrypoint: "lib/servers/cron.js"
-    schedule: "0 2 * * *"
-  - name: balance-notif
-    appEntrypoint: "lib/servers/daily-balance-notification.js"
-    schedule: "0 14 * * *"
-mongobackup:
-  bucketName: galoy-staging-backups
-  mongoToolsVersion: 4.2.14-r6
-  image:
-    repository: us.gcr.io/galoy-org/mongo-backup
-    digest: "sha256:4cfc197ea082ef3dae2d4321a9dfa31ef185f43d578a37a224095b75de656e6a"
-    git_ref: 872b70e
-loglevel: debug
-needFirebaseServiceAccount: true
-twilio: true
-# value can be set here, or override with helm,
-# ie: helm upgrade -i --set twilio.TWILIO_PHONE_NUMBER=6505550001,TWILIO_ACCOUNT_SID=AKBQWN...
-# twilio:
-#   TWILIO_PHONE_NUMBER: "phone"
-#   TWILIO_ACCOUNT_SID: "sid"
-#   TWILIO_AUTH_TOKEN: "authtoken"
-redis:
-  master:
-    persistence:
+
+    ## Service type
+    ##
+    serviceType: ClusterIP
+
+    ## Liveness/Readiness Probes Configuration
+    ## Determines if pod is healthy or if it should be killed.
+    ## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/
+    ##
+    probes:
+      ## Enable/disable Admin API probes for healthcheck
+      enabled: true
+
+      ## If pod is starting up/healthy
+      liveness:
+        initialDelaySeconds: 60
+        periodSeconds: 10
+        failureThreshold: 5
+        timeoutSeconds: 1
+
+      ## When to expose the pod to the service
+      readiness:
+        initialDelaySeconds: 5
+        failureThreshold: 5
+        successThreshold: 2
+        timeoutSeconds: 1
+
+    ## Galoy database backup configuration
+    ## This is a custom backup configuration, which means data lying in
+    ## MongoDB and LND can be backed up by custom scripts present in the
+    ## galoy codebase itself.
+    ##
+    backups:
+
+      ## Backup data to Google Cloud Storage Bucket
+      ## To backup to GCS bucket, you would need a GCP Service Account
+      ## that lets you upload to a specific bucket
+      ##
+      gcs:
+        # Enable/disable uploading backup to GCS bucket
+        enabled: false
+
+        # Configuration for GCS bucket
+        config:
```

Comment:
I don't think we need a `config` key for the gcs or dropbox backups. It's adding an additional level of nesting, which imo isn't needed

Reply:
We need this to specify the GCS bucket name

Reply:
why can't we specify the gcs bucket name like:
```
gcs:
  enabled: true
  bucketName: xyz
```

Reply:
Ah, okay I will incorporate this.

Reply:
Done